packages/material-ui/src/Icon/Icon.js

```diff
@@ -66,6 +66,8 @@ const Icon = React.forwardRef(function Icon(props, ref) {
     <Component
       className={clsx(
         'material-icons',
+        // Prevent translations of the content which breaks the CSS logic.
```

Comment:
One nit before merge since I often see these kinds of comments in our codebase: What is "the CSS logic"? As far as I can tell translating simply brakes font based icons.

Reply:
```suggestion
        // Prevent translations of the content.
        // The font relies on the exact text content to render the shape.
```